spec/line/bot/v2/misc_spec.rb

```diff
@@ -199,4 +199,39 @@
       expect(body._next).to eq("token")
     end
   end
+
+  describe 'POST /v2/bot/message/push' do
```

Comment:
Even without this change in `lib/line/bot.rb`, the test passes. This means that the test cannot detect the same failure you encountered. Can you write a test that fails without the change  in `lib/line/bot.rb` and succeeds with the patch in `lib/line/bot.rb`?